test_app/tests/lib/utils/test_timeout.py

```diff
@@ -0,0 +1,16 @@
+from time import sleep
+
+import pytest
+
+from ansible_base.lib.utils.timeout import Timeout
+
+
+def test_timeout_success():
```

Comment:
Maybe use `time.time()` to make sure this doesn't take 10 seconds when it shouldn't.

Reply:
This is going away.